apps/build.info

```diff
@@ -23,6 +23,11 @@ IF[{- !$disabled{apps} -}]
   INCLUDE[openssl]=.. ../include
   DEPEND[openssl]=libapps.a ../libssl
 
+IF[{- $config{target} =~ /^(?:Cygwin|mingw|VC-)/ -}]
+  GENERATE[openssl.rc]=util/mkrc.pl openssl
+  SOURCE[openssl]=openssl.rc
+ENDIF
```

Comment:
Appveyor shows us that this creates a "surprising" disturbance...  [this section in apps/progs.pl](https://github.com/openssl/openssl/blob/master/apps/progs.pl#L24-L26) needs to be rewritten by inserting a grep in the middle, like this:

``` perl
diff --git a/apps/progs.pl b/apps/progs.pl
index f832467b57..f16bc10fac 100644
--- a/apps/progs.pl
+++ b/apps/progs.pl
@@ -23,6 +23,7 @@ my $apps_openssl = shift @ARGV;
 # the lookups in %unified_info
 my @openssl_source =
     map { @{$unified_info{sources}->{$_}} }
+    grep { /\.o$/ }
         @{$unified_info{sources}->{$apps_openssl}};
 
 foreach my $filename (@openssl_source) {
```

Reply:
done, and also updated copyright year generation.  please reconfirm.